platform/systemd/kanidmd.service

```diff
@@ -1,20 +1,15 @@
-# Source: https://build.opensuse.org/package/view_file/home:firstyear:kanidm/kanidm/kanidmd.service
-# You should not need to edit this file. Instead, use a drop-in file as described in:
-#   /usr/lib/systemd/system/kanidmd.service.d/custom.conf
-
 [Unit]
 Description=Kanidm Identity Server
 After=chronyd.service ntpd.service network-online.target
 Before=radiusd.service
 
 [Service]
-Type=simple
-DynamicUser=yes
```

Comment:
DynamicUser is required, this change is not acceptable.

Ref: https://unix.stackexchange.com/a/635036 / http://0pointer.net/blog/dynamic-users-with-systemd.html

Reply:
Agreed.